login.css

```diff
@@ -195,7 +194,7 @@ p{
 	color:black;
 	font-size:14px;
 	text-decoration:none;
-	font-family: 'Roboto', sans-serif;
+	font-family: 'Open Sans', sans-serif;
```

Comment:
20/08/2021: Changing the fonts style for the update(forgot password link) to be  Open Sans.